package.json

```diff
@@ -137,6 +137,7 @@
     "chai-as-promised": "^7.1.1",
     "chai-jest-snapshot": "^2.0.0",
     "codecov": "^3.8.2",
+    "commander": "^8.3.0",
```

Comment:
Introduced this one to have an easy way to parse command-line args

Reply:
We used to have `argparse` in some scripts. Would be great to consolidate.

Maybe a small RFC to explain the choice?